templates/openstack/install.html

```diff
@@ -41,1176 +41,98 @@ <h1>Install OpenStack yourself</h1>
-  <div class="p-stepped-list__content">
-    <p>Set the admin user's password for your OpenStack cluster by executing the following command:</p>
-    <div class="p-code-copyable">
-      <input class="p-code-copyable__input"
-      value="juju config keystone admin-password=<password>" readonly="readonly">
-        <button class="p-code-copyable__action">Copy to clipboard</button>
-      </div>
-      <p>The Horizon service is available at the IP address displayed by running:</p>
-      <div class="p-code-copyable">
-        <input class="p-code-copyable__input"
-        value="juju status | grep openstack-dashboard | tail -n 1 | awk '{print $5}'" readonly="readonly">
-        <button class="p-code-copyable__action">Copy to clipboard</button>
-      </div>
-      <p>Go to <code>http://&lt;Horizon IP&gt;:80/horizon</code> URL. You should see the OpenStack login screen. Type your credentials and press the "Sign In" button. Use "admin_domain" in the "Domain" field:</p>
-      <p>
-        <a href="https://assets.ubuntu.com/v1/88a77f0d-openstack+dashboard+login.png">
-          {{
-            image(
-            url="https://assets.ubuntu.com/v1/88a77f0d-openstack+dashboard+login.png",
-            alt="",
-            height="241",
-            width="424",
-            hi_def=True,
-            loading="lazy",
-            ) | safe
-          }}
-        </a>
-      </p>
-      <p>You should be able to see the OpenStack welcome screen. Click on the "admin" drop-down list in the top right corner and press "OpenStack RC File":</p>
-      <p>
-        <a href="https://assets.ubuntu.com/v1/1dd72150-openstack+admin+click.png">
-          {{
-            image(
-            url="https://assets.ubuntu.com/v1/1dd72150-openstack+admin+click.png",
-            alt="",
-            height="241",
-            width="424",
-            hi_def=True,
-            loading="lazy",
-            ) | safe
-          }}
-        </a>
-      </p>
-      <p>Some browsers may require that you confirm that you want to download this file. If you are using Google Chrome, press "Keep" from the menu on the bottom:</p>
-      <p>
-        <a href="https://assets.ubuntu.com/v1/87387fd1-openstack+click+keep.png">
-          {{
-            image(
-            url="https://assets.ubuntu.com/v1/87387fd1-openstack+click+keep.png",
-            alt="",
-            height="241",
-            width="424",
-            hi_def=True,
-            loading="lazy",
-            ) | safe
-          }}
-        </a>
+<section class="p-strip is-bordered" id="large-scale-deployment">
+  <div class="row u-equal-height">
+    <h2>Large-scale deployment</h2>
+    <p>
+      These instructions use <a href="https://docs.openstack.org/charm-guide/latest/">OpenStack Charms</a>. OpenStack Charms are the foundation of Canonical’s <a href="https://ubuntu.com/openstack">Charmed OpenStack</a> distribution which is an enterprise private cloud, designed to run mission-critical workloads.
```

Comment:
```suggestion
      These instructions use <a href="https://docs.openstack.org/charm-guide/latest/">OpenStack Charms</a>. OpenStack Charms are the foundation of Canonical’s <a href="https://ubuntu.com/openstack">Charmed OpenStack</a> distribution which is an enterprise private cloud, designed to run mission-critical workloads
```
Remove full-stop to keep consistent